### PR TITLE
adds support for engines

### PR DESCRIPTION
**Pull request by jbailey4**
Moves the component's templates from the `app/templates/components` into the `addon/templates/components` directory, allowing engines to render properly.

fixes https://github.com/yapplabs/ember-radio-button/issues/45

### CODE REVIEW

File: addon/components/radio-button.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import Ember from 'ember';
+import layout from '../templates/components/radio-button';
```
Comment (raycohen) [inline addon/components/radio-button.js]: is there a way to have these paths not be relative to the current file?

**Comment by raycohen** [comment]
is there a way we can test this?  Maybe we should add an in-repo engine that gets used in the dummy app?

File: addon/components/radio-button.js (hunk 1 above)
Comment (jbailey4) [inline addon/components/radio-button.js]: @raycohen This is what ember-cli gives you by default when you generate a component: `ember g component radio-button`. I'm not sure if you can use the addon's namespace. I can try.

**Comment by jbailey4** [comment]
@raycohen Yeah the in-repo engine sounds good, I can add that if you like. I tested using a standalone engine and npm linking this addon.

**Comment by raycohen** [comment]
Yeah, add the in-repo engine and I think we can merge this and do a release. Thanks!

**Comment by raycohen** [comment]
oh, and I've updated master to use ember-cli 2.9, so you should rebase

**Comment by jbailey4** [comment]
@raycohen Update: I added the in-repo engine to the project, but ran into an issue with the engine consumed by the test dummy app resolving the `radio-button` component. I'm not exactly sure how an in-repo engine differs within an addon vs. an app, but I'm 💯  this works with the latter. Also works nested within an addon, which is consumed by the engine and the host app, in this case both the host app and the engine has access to the components.

**Comment by raycohen** [comment]
thanks @jbailey4 and sorry for the delay. 
published version 1.1.1 - can you verify that it works in your setup?

**Comment by jbailey4** [comment]
@raycohen Thanks and no problem. Will let you know if any issues.

**Pull request merged**